include/certifier_framework.h

```diff
@@ -336,6 +336,7 @@ class cc_trust_manager {
   bool certify_secondary_domain(const string &domain_name);
   bool get_certifiers_from_store();
   bool put_certifiers_in_store();
+  bool write_private_key_to_file(const string &filename);
```

Comment:
Need to externalize this method, so we can SWIG it. This will persist the private-key to a file, so that Python SSL certificate handling interfaces can use that file directly.

Reply:
So this will become a byte stream, right?